tests/vppcallmock/vpp_ctx_mock.go

```diff
@@ -68,16 +68,23 @@ type mockedChannel struct {
 	Msgs []govppapi.Message
 }
 
-//SendRequest just save input argument to structure field for future check
+// SendRequest just save input argument to structure field for future check
 func (m *mockedChannel) SendRequest(msg govppapi.Message) *govppapi.RequestCtx {
 	m.Msg = msg
 	m.Msgs = append(m.Msgs, msg)
 	return m.channel.SendRequest(msg)
 }
 
-//SendMultiRequest just save input argument to structure field for future check
+// SendMultiRequest just save input argument to structure field for future check
 func (m *mockedChannel) SendMultiRequest(msg govppapi.Message) *govppapi.MultiRequestCtx {
 	m.Msg = msg
 	m.Msgs = append(m.Msgs, msg)
 	return m.channel.SendMultiRequest(msg)
 }
+
+// CheckMessageCompatibility checks whether provided messages are compatible with the version of VPP
+// which the library is connected to
+func (m *mockedChannel) CheckMessageCompatibility(msgs ...govppapi.Message) error {
+	m.Msgs = msgs
```

Comment:
The field `Msgs` is used in `SendRequest`/`SendMultiRequest` for list of sent requests, how is it relevant here?